### DISCUSSION TITLE
How do I translate a box so that it does not intersect a different box?

### DISCUSSION THREAD

**Discussion by jburz2001** [nrc-cnrc/EGSnrc#1137] [Questions] [answered]
Hello. I wrote an input file which yields the following visualization: 
![image](https://github.com/nrc-cnrc/EGSnrc/assets/56214083/24d1debc-73e0-4481-805e-3d2b1eb7b8ec)

However, I actually want the thin box to be translated along the y-axis so that it is above the rotated red box. And of course, I would like the pencil beam source to be outside of these geometries. How do I do this? I've been trying to use the egs_gtransformed library to handle the translations / rotations along with the egs_gunion library to use both boxes. What am I doing wrong? My code is attached below.

[comet_450_minifocus.egsinp.zip](https://github.com/nrc-cnrc/EGSnrc/files/15501623/comet_450_minifocus.egsinp.zip)

**Comment by rtownson** [comment, marked as answer]
First, create a big air box for everything to be inside. Use egs_genvelope to put other geometries inside the box. As long as your boxes don't overlap or touch, you can just put them both in the envelope.

Also, you just have a typo, you used `:start translation:` instead of `:start transformation:`.

**Reply by jburz2001** [reply]
Thank you for your help. This fixed the confusions I was having.
